js/admin/carrier_wizard.js

```diff
@@ -792,3 +802,16 @@ function checkAllZones(elt)
 	}
 
 }
+
+/**
+ * Check range validation
+ * @param index
+ */
+function checkCarriersRange(index)
+{
+	if (validateRange(index)) {
+		enableRange(index);
+	} else {
+		disableRange(index);
+	}
+}
```

Comment:
All new methods should be "namespaced" (that is, not in the global scope).

One "namespacing" technique is to make functions be properties of a namespace variable (see the example I provided before).